spec/javascripts/fixtures/NewsletterSticky.html

```diff
@@ -1,9 +1,11 @@
 <div id="fixture-1">
   <div class="news-signup-sticky news-signup-sticky--positioned" data-dough-component="NewsletterSticky">
     <form onsubmit="return false;">
-      <button class="news-signup-sticky__close">
+      <button class="news-signup-sticky__close" id="first-focusable">
```

Comment:
I understand that t- represents test and shouldn't really be used for styling. Often these classes shouldn't really still be there but have tended to remain after the testing has occurred.